server/network/aoprotocol.py

```diff
@@ -549,31 +591,36 @@ def net_cmd_ms(self, args):
                 self.client.send_ooc('You don\'t any areas!')
                 return
             text = ' '.join(part[1:])
+            send_args[4] = text
         elif text.startswith('/testify '): # Start a new testimony in this area.
             part = text.split(' ')
             text = ' '.join(part[1:]) # remove command
+            send_args[4] = text
             if not self.client.area.start_testimony(self.client, text):
                 return
             text = '~~-- ' + text + ' --'
+            send_args[4] = text
             color = 3 # orange
         elif text.startswith('/examine'): # Start an examination of this area's testimony.
             if not self.client.area.start_examination(self.client):
                 return
             text = '~~-- ' + self.client.area.testimony.title + ' --'
+            send_args[4] = text
```

Comment:
why is this line meaningful if nothing is done with send_args here, it will be overshadowed by a new send_args below